rest_api/operator_apis/dns-operator-openshift-io-v1.adoc

```diff
@@ -127,8 +127,8 @@ Type::
 Description::
 +
 --
-tolerations is a list of tolerations applied to DNS pods. 
- If empty, the DNS operator sets a toleration for the "node-role.kubernetes.io/master" taint.  This default is subject to change.  Specifying tolerations without including a toleration for the "node-role.kubernetes.io/master" taint may be risky as it could lead to an outage if all worker nodes become unavailable. 
+tolerations is a list of tolerations applied to DNS pods.
+ If empty, the DNS operator sets a toleration for the "node-role.kubernetes.io/master" taint.  This default is subject to change.  Specifying tolerations without including a toleration for the "node-role.kubernetes.io/master" taint may be risky as it could lead to an outage if all worker nodes become unavailable.
```

Comment:
Extra spaces before sentences.

Use "might" rather than "may, and "because" rather than "as": "...taint might be risky because it could lead to" IBM SG p. 276